<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControleHotel</class>
 <widget class="QMainWindow" name="ControleHotel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Controle do Hotel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="accessControlButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Controle de Acesso</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reserveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reserva</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clientControlButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Controle de Clientes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reportButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Relatórios</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>391</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="controlAccessPage">
     <widget class="QLabel" name="controlAccessPageLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tela de Controle de Acesso</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="reservePage">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>381</width>
        <height>331</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="create">
       <property name="accessibleName">
        <string/>
       </property>
       <attribute name="title">
        <string>Criar</string>
       </attribute>
       <widget class="QLabel" name="clientReserveLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>65</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Cliente</string>
        </property>
       </widget>
       <widget class="QPushButton" name="concludeReserveButton">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>240</y>
          <width>181</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Concluir</string>
        </property>
       </widget>
       <widget class="QComboBox" name="clientReserveComboBox">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>241</width>
          <height>24</height>
         </rect>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertPolicy::InsertAlphabetically</enum>
        </property>
       </widget>
       <widget class="QLabel" name="roomReserveLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>65</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Quarto</string>
        </property>
       </widget>
       <widget class="QComboBox" name="roomReserveComboBox">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>241</width>
          <height>24</height>
         </rect>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertPolicy::InsertAlphabetically</enum>
        </property>
       </widget>
       <widget class="QLabel" name="periodReserveLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>65</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Período</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="initialReserveDateEdit">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>140</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="locale">
         <locale language="Portuguese" country="Brazil"/>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2024</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="minimumDate">
         <date>
          <year>2024</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="displayFormat">
         <string>dd/M/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QDateEdit" name="endReserveDateEdit">
        <property name="geometry">
         <rect>
          <x>239</x>
          <y>140</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="minimumDate">
         <date>
          <year>2024</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="displayFormat">
         <string>dd/M/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="aReserveLabel">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>140</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>a</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Remover</string>
       </attribute>
       <widget class="QComboBox" name="removeReservationComboBox">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>70</y>
          <width>241</width>
          <height>24</height>
         </rect>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertPolicy::InsertAlphabetically</enum>
        </property>
       </widget>
       <widget class="QLabel" name="RemoveReservationLabel">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>50</y>
          <width>57</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Reservas</string>
        </property>
       </widget>
       <widget class="QPushButton" name="removeReservationButton">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>210</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Remover</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="clientsControlPage">
     <widget class="QTabWidget" name="clientsContrlTabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>381</width>
        <height>331</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="create_3">
       <property name="accessibleName">
        <string/>
       </property>
       <attribute name="title">
        <string>Criar</string>
       </attribute>
       <widget class="QLineEdit" name="nameControlClientLineEdit">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>60</y>
          <width>181</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="documentControlClientLineEdit">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>110</y>
          <width>181</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="addressClientControlLineEdit">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>150</y>
          <width>181</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="cellphoneControlClientLineEdit">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>190</y>
          <width>181</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="nameControlClientLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>65</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nome</string>
        </property>
       </widget>
       <widget class="QLabel" name="documentControlClientLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Documento</string>
        </property>
       </widget>
       <widget class="QLabel" name="addressClientControlLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>160</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Endereço</string>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::AutoText</enum>
        </property>
       </widget>
       <widget class="QLabel" name="cellphoneClientControlLineEdit">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>200</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Telefone</string>
        </property>
       </widget>
       <widget class="QPushButton" name="concludeControlClientButton">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>240</y>
          <width>181</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Concluir</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="reportPage">
     <widget class="QLabel" name="reportPageLabel">
      <property name="geometry">
       <rect>
        <x>104</x>
        <y>80</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Tela de Relatórios</string>
      </property>
      <property name="buddy">
       <cstring>reportPageLabel</cstring>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionTeste_1">
   <property name="text">
    <string>Teste 1</string>
   </property>
  </action>
  <action name="actionTeste_2">
   <property name="text">
    <string>Teste 2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
